### ISSUE TITLE
Failing now for old versions of Raku

### ISSUE THREAD

**Issue by JJ** [Leont/getopt-long6#11]
I haven't found out which ones yet, but it looks like pretty much all of them are failing with something like:

```
===> Testing: Getopt::Long:ver<0.3.1>
[Getopt::Long] # Failed test 'Parse a custom parseable type'
[Getopt::Long] # at t/basic.t line 140
[Getopt::Long] # expected: Foo.new(value => 1)
[Getopt::Long] # got: "1"
[Getopt::Long] # Failed test 'right exception type (Getopt::Long::Exception)'
[Getopt::Long] # at SETTING::src/core.c/Exception.pm6 line 62
[Getopt::Long] # Expected: Getopt::Long::Exception
[Getopt::Long] # Got: X::Method::NotFound
[Getopt::Long] # Exception message: No such method 'Signature' for invocant of type 'Str'
[Getopt::Long] # You failed 1 test of 2
[Getopt::Long] # Failed test 'No conversion known for type Signature'
[Getopt::Long] # at t/basic.t line 142
[Getopt::Long] # You failed 2 tests of 43
===> Testing [FAIL]: Getopt::Long:ver<0.3.1>
```

This is probably just as well, and totally intended, but I re-build Docker containers with every new release, and this is failing now. I can use the old versions from the ecosystem, so this is not such a big deal. It's just FYI.

**Comment by Leont** [comment]
That should be fixed in 0.3.2. It was testing testing with conversion type functionality that isn't available before 2020.11, so on older versions it now just skips those tests.

**Comment by JJ** [comment]
Still failing, although in a different way, for 2019.07.1

```
[Getopt::Long] ===SORRY!===
[Getopt::Long] Could not locate compile-time value for symbol CORE::Exception
```
